flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogTest.java

```diff
@@ -44,82 +44,96 @@ public static void init() throws IOException {
 	// =====================
 
 	// TODO: re-enable these tests once HiveCatalog support table operations
-	@Test
```

Comment:
I'm wondering what's the point of removing test annotation as those tests are empty anyway.